st2common/st2common/content/utils.py

```diff
@@ -138,6 +139,32 @@ def get_pack_base_path(pack_name):
     return pack_base_path
 
 
+def get_pack_directory(pack_name):
+    """
+    Retrieve a directory for the provided pack.
+
+    If a directory for the provided pack doesn't exist in any of the search paths, None
+    is returned instead.
+
+    Note: If same pack exists in multiple search path, path to the first one is returned.
+
+    :param pack_name: Pack name.
+    :type pack_name: ``str``
+
+    :return: Pack to the pack directory.
+    :rtype: ``str`` or ``None``
+    """
+    packs_base_paths = get_packs_base_paths()
+    for packs_base_path in packs_base_paths:
+        pack_base_path = os.path.join(packs_base_path, pipes.quote(pack_name))
```

Comment:
I meant os.path.join(packs_base_path, pipes.quote(pack_name) would return pack_path and not packs_base_path :P. Sorry.